Makefile

```diff
@@ -22,14 +29,45 @@ gofmt:
 $(GOLINT):
 	go install golang.org/x/lint/golint
 
+$(GEN_VALUEWRAPPER): $(wildcard ./internal/gen-valuewrapper/*)
+	go build -o $@ ./internal/gen-valuewrapper
+
+$(GEN_ATOMICINT): $(wildcard ./internal/gen-atomicint/*)
+	go build -o $@ ./internal/gen-atomicint
+
 .PHONY: golint
 golint: $(GOLINT)
 	$(GOLINT) ./...
 
 .PHONY: lint
-lint: gofmt golint
+lint: gofmt golint generatenodirty
+
+# Hack to reference " " in subst
+space :=
+space +=
+
+# comma separated list of packages to consider for code coverage.
+COVER_PKG = $(subst $(space),1,$(shell \
```

Comment:
since we're already running a shell pipeline, should we use `tr` to replace space with `,` rather than dealing with make?

Reply:
Good call. Switching.

Reply:
Actually, `tr` will add a trailing comma. Going to use [`paste`](https://www.gnu.org/software/coreutils/manual/html_node/paste-invocation.html) instead.

    $ echo -n -e 'foo\nbar\nbaz\n' | paste -sd, -
    foo,bar,baz